README.md

```diff
@@ -145,8 +146,28 @@ The input's file paths and directory structure will be preserved in the [`dag-pb
 - `blockWriteConcurrency` (positive integer, defaults to 10) How many blocks to hash and write to the block store concurrently. For small numbers of large files this should be high (e.g. 50).
 - `fileImportConcurrency` (number, defaults to 50) How many files to import concurrently. For large numbers of small files this should be high (e.g. 50).
 
+## Overriding internals
+
+Several aspects of the importer are overridable by specifying functions as part of the options object with these keys:
```

Comment:
It would be useful to know the role of each of these in addition to their signature and return type. It’s great that they can be overridden but I don’t see why/how a user can use it without knowing what part these functions play in the import process.